### PR TITLE
fix: Authorisation fixes

### PR DESCRIPTION
**Pull request by iamleeon**
- Renamed "hashed_password" to "password" on signup.
- Renamed "username" to "email" on login.
- Changed sqlalchemy.url  in alembic.ini to resolve issue with empty db.
- other small fixes and adjustments.

### CODE REVIEW

**Review by oleksyii — CHANGES_REQUESTED** [review]
PLEASE FIX docker-compose.yml!!!

File: docker-compose.yml [hunk 1]
```diff
@@ -20,10 +20,13 @@ services:
     depends_on:
       db-app:
         condition: service_healthy
-      redis:
-        condition: service_started
```
[the remark this answers is not in the document]
Comment (iamleeon) [reply docker-compose.yml:24]: I have no idea how it happened. I didn't remove anything from this file. Thanks for pointing it out. Corrected.

File: docker-compose.yml [hunk 2]
```diff
@@ -48,15 +52,6 @@ services:
...
 
-  redis:
-    image: redis:alpine
-    container_name: redis_app
-    ports:
-      - "6379:6379"
-    networks:
-      - app-network
-    restart: unless-stopped
-
```
[the remark this answers is not in the document]
Comment (iamleeon) [reply docker-compose.yml:59]: Thanks for pointing it out. Corrected.

File: alembic.ini [hunk 3]
```diff
@@ -61,8 +61,8 @@ version_path_separator = os  # Use os.pathsep. Default configuration used for ne
 # are written from script.py.mako
 # output_encoding = utf-8
 
-sqlalchemy.url = driver://user:pass@localhost/dbname
-
+# sqlalchemy.url = driver://user:pass@localhost/dbname
+sqlalchemy.url = postgresql+psycopg2://${POSTGRES_USER}:${POSTGRES_PASSWORD}@db-app:5432/${POSTGRES_DB}
```
[the remark this answers is not in the document]
Comment (iamleeon) [reply alembic.ini:65]: I faced an issue where tables didn't load into the database after running our docker containers (or locally). This is the only possible fix I could come up with. `sqlalchemy.url = driver://user:pass@localhost/dbname` didn't work for me.

**Review by oleksyii — APPROVED** [review]

**Pull request merged**